### PR TITLE
print reloc symbol address

### PR DESCRIPTION
**Pull request by mytbk**

### CODE REVIEW

File: libr/core/disasm.c [hunk 1]
```diff
@@ -3773,7 +3773,8 @@ static void ds_print_relocs(RDisasmState *ds) {
 		if (rel->import) {
 			r_cons_printf ("; RELOC %d %s", rel->type, rel->import->name);
 		} else if (rel->symbol) {
-			r_cons_printf ("; RELOC %d %s", rel->type, rel->symbol->name);
+			r_cons_printf ("; RELOC %d %s @ 0x%llx + 0x%llx", rel->type,
```
Comment (radare) [inline libr/core/disasm.c]: use PFMT64x, llx is not portable, also pad with %08

Comment (radare) [reply libr/core/disasm.c]: one of the offsets is duplicated

Comment (ret2libc) [reply libr/core/disasm.c]: what do you mean by duplicated? where?

**Comment by radare** [comment]
this comment is printed in one location, so the offset of this location doesnt nees to be duplicated

**Comment by ret2libc** [comment]
That's what I thought at first, but I checked it out and it wasn't.
See here:

```
[0x0023c8e8]> pd 1
            ;-- reloc.fuse_fs_read_buf_232:
            0x0023c8e8      .qword 0x0000000000007826                  ; RELOC 64 fuse_fs_read_buf @ 0xd380 + 0x0
[0x0023c8e8]> pd 3 @ 0xd380
            ;-- fuse_fs_read_buf:
            0x0000d380      4157           push r15
            0x0000d382      4156           push r14
            0x0000d384      4d89c7         mov r15, r8
```

**Comment by radare** [comment]
oh ok, i see. and also maybe better not to pad with zeroes the delta

thx for the snippet

File: libr/core/disasm.c (hunk 1 above)
Comment (radare) [reply libr/core/disasm.c]: i will merge if you use PFMT64x instead of llx, pls update

**Comment by mytbk** [comment]
updated

**Pull request merged**